src/mixins/rows.js

```diff
@@ -70,6 +70,7 @@ export default {
                     groupColumn,
                     selected: false,
                     originalIndex: index,
+                    columnsThatExactlyMatchesFilter: [],
```

Comment:
columns (plular) => "Match"
We could consider a shorter name too: `exactMatchColumns`. This pairs well with `exactMatch` row property.